### ISSUE TITLE
[Bug report] no attribute bonus to dmg/def

### ISSUE THREAD

**Issue by wereant** [Pyrdacor/Ambermoon.net#303]
Strength 25 shall give +1 to dmg, but character screen doesn't show this. With no weapon equipped dmg=0 , with a weapon, say, +5 dmg=5. No bonus from strength. Same with stamina and defence stat

**Comment by Pyrdacor** [comment]
Thanks for reporting. I will have a look at it soon.

**Comment by Pyrdacor** [comment]
I accidently only added the influence of STR/STA if the attack or protection buffs are active. I fixed it now. Will be part of the next release.

**Comment by wereant** [comment]
Thank you very much! Is it normal that charge item (from scroll) added only one charge to the crystall ball? Also, when i tried to identify already identified item, game crashed (spell from scroll, used by thief).

**Comment by Pyrdacor** [comment]
As far as I remember it really gives only 1 charge. But I will have a look at the original code.

I'll create a new issue for the crash now.

**Comment by wereant** [comment]
Oh sorry to bother you. I just read spell description, it says: Adds n charges to a magic item where n is a random number between 1 and the minimum of caster level and max possible charges. Case closed

**Comment by Pyrdacor** [comment]
I looked at the code and also tested it. Charge item (from scroll or not) grants a random value between 1 and the minimum of caster level and possible charges.

So if you have a level 15 thief it can add 1 to 15 charges at max. But only as many as the item can hold.

I tested with Mando (thief) and the scroll. I had a crystal ball with 24 charges and used the scroll. It had 48 charges afterwards. Maybe just bad luck on your end?

I also couldn't reproduce the crash. Tested many times to identify already identified items and it worked without problems. From scroll or via spell. Did you identify a particular item maybe?
